docs/why-rojo.md

```diff
@@ -16,30 +18,33 @@ These advanced text editors have features like multi-cursor editing, go-to symbo
 
 Many Rojo VS Code users also use extensions like:
 
-* [vscode-rbxlua](https://marketplace.visualstudio.com/items?itemName=AmaranthineCodices.vscode-rbxlua)
-* [Roblox Lua Autocompletes](https://marketplace.visualstudio.com/items?itemName=Kampfkarren.roblox-lua-autofills)
-* [TabNine](https://tabnine.com)
+- [vscode-rbxlua](https://marketplace.visualstudio.com/items?itemName=AmaranthineCodices.vscode-rbxlua)
+- [Roblox Lua Autocompletes](https://marketplace.visualstudio.com/items?itemName=Kampfkarren.roblox-lua-autofills)
```

Comment:
Replace with Roblox-LSP?

Reply:
I'm torn on if we should make these changes in this PR - I agree with all of the changes you propose but just in case someone has an issue w/ them I think we should wait it out till this is approved